activation/atxdb.go

```diff
@@ -635,6 +648,11 @@ func (db *DB) GetEpochAtxs(epochID types.EpochID) (atxs []types.ATXID) {
 		}
 		atxs = append(atxs, a)
 	}
+	atxIterator.Release()
+	err := atxIterator.Error()
```

Comment:
You can collapse this line and the next line into one line, e.g., `if err := atxIterator.Error(); err != nil {`